src/API/Authentication.php

```diff
@@ -521,12 +528,16 @@ public function oauth_token($options = [])
             throw new ApiException('grant_type is mandatory');
         }
 
-        return $this->apiClient->post()
-        ->oauth()
-        ->token()
-        ->withHeader(new ContentType('application/json'))
```

Comment:
Where is this now set?

Reply:
`method('post')` takes care of that automatically. `post()` here is a magic method, which I'll slowly pruning out.